diff --git a/ports/gmp/portfile.cmake b/ports/gmp/portfile.cmake
index 1d0020bb5..e6c3ade91 100644
--- a/ports/gmp/portfile.cmake
+++ b/ports/gmp/portfile.cmake
@@ -19,6 +19,8 @@ vcpkg_download_distfile(
     SHA512 4a7c50dc0a78e6c297c0ac53129ed367dbf669100a613653987d0eddf175376296254ed26ecce15d02b0544b99e44719af49635e54982b22e745f22e2f8d1eda
 )
 
+set(VCPKG_BUILD_TYPE release)
+
 vcpkg_extract_source_archive(SOURCE_PATH
     ARCHIVE "${ARCHIVE}"
     SOURCE_BASE "v${VERSION}"
@@ -28,6 +30,7 @@ vcpkg_extract_source_archive(SOURCE_PATH
         subdirs.patch
         msvc_symbol.patch
         arm64-coff.patch
+
         "${ARM64PATCH}"
 )
 
@@ -120,3 +123,5 @@ vcpkg_install_copyright(
         "${SOURCE_PATH}/COPYINGv3"
         "${SOURCE_PATH}/COPYINGv2"
 )
+
+vcpkg_build_make(BUILD_TARGET check SUBPATH tests)
